lib/sdf/link.rb

```diff
@@ -19,6 +19,16 @@ def pose
             Conversions.pose_to_eigen(xml.elements["pose"])
         end
 
+        # Check if link is kinematic
+        #
+        #
+        def kinematic
```

Comment:
The convention in Ruby is that predicate methods end with '?', i.e. `def kinematic?`